plugin/pkg/scheduler/generic_scheduler.go

```diff
@@ -140,16 +140,29 @@ func (g *genericScheduler) selectHost(priorityList schedulerapi.HostPriorityList
 		return "", fmt.Errorf("empty priorityList")
 	}
 
-	sort.Sort(sort.Reverse(priorityList))
-	maxScore := priorityList[0].Score
-	firstAfterMaxScore := sort.Search(len(priorityList), func(i int) bool { return priorityList[i].Score < maxScore })
+	// extract nodes with highest score
+	// TODO(harry) should we refactor PrioritizeNodes into return nodes with max score only?
```

Comment:
I don't think so. We shouldn't change the contract of PrioritizeNodes (at least for now).